.storybook/preview.js

```diff
@@ -9,13 +10,65 @@ import 'focus-visible';
 import '../src/index.scss';
 import './preview.scss';
 
+/**
+ * @typedef { [String, import('@storybook/client-api/dist/types').StoreItem] } StoryItem
+ */
+
 // Accessibility testing via aXe
 addDecorator(withA11y);
 
 // Theme selection from stories
 const themes = [{ name: 'Dark', class: 't-dark', color: colors.primaryBrand }];
 addParameters({ themes });
 
+/**
+ * Get the category from a `storySort` story
+ * @param {StoryItem} story - Item passed to `addParameters.options.storySort`
+ * @returns {string} - Story's category, the first part of the `StoreItem.kind`
+ * value, which is a string delimited with forward slashes
+ */
+const getStoryCategory = (story) => story[1].kind.split('/')[0];
+
+/**
+ * Define an ordered list of story categories. These should match the first part
+ * of the `<Meta>` component's `title` attribute found in each story (`.mdx` file)
+ */
+const orderedCategories = [
+  'Getting Started',
+  'Design',
+  'Design Tokens',
+  'Objects',
+  'Components',
+  'Utilities',
+  'Themes',
+  'Third-Party',
+];
+/**
```

Comment:
This code could be simplified by getting rid of the `sanitize` functionality, but I thought it might be a good idea. Up for review.